admin-portal/src/app/(pages)/signin/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client"
+// import axios, { AxiosResponse } from "axios";
+import React, { useState } from 'react';
+import QRCode from 'react-qr-code';
+
+export default function EventDetailsPage() {
+  const [eventId, setEventId] = useState("");
+  const [eventDetails, setEventDetails] = useState({
+    id: "",
+    name: "",
+    description: "",
+    org: "",
+  });
+
+  const handleGenerateQRCode = () => {
+    // Simulate fetching event details using the entered event ID
+    const fetchedDetails = {
+      id: eventId,
+      name: `Event for ${eventId}`,
+      description: `Description for event ${eventId}`,
+      org: "Sample Organization",
+    };
+    setEventDetails(fetchedDetails);
+  };
+
+  return (
+    <div style={{ textAlign: "center", margin: "20px" }}>
```

Comment:
Move this and all styles to a dedicated EventSignIn.css file and import the file to access the styles.